_episodes/009-jan-lucio.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Double Feature: Jan-Erik Rediger on RustFest & Lucio Franco on the Tonic gRPC framework"
+date: 2020-01-06T19:00:00+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e009-rustfest-jan-erik-rediger.mp3
+duration: "25:59"
+length: "18710674"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+Two more interviews from RustFest 2019, first with lead RustFest organizer Jan-Erik Rediger and second with Tokio contributor Lucio Franco on the Tower gRPC framework.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to be interviewed, propose a topic for an episode, or help with hosting or audio editing!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+# [@00:00] Part 1: RustFest w/ Jan-Erik Rediger
```

Comment:
This is probably going to show up weirdly on the website, but I can fix that separately :)